src/main/java/com/amazonaws/services/kinesis/leases/impl/LeaseManager.java

```diff
@@ -620,7 +620,7 @@ public void updateLeaseWithMetaInfo(T lease, UpdateField updateField)
         UpdateItemRequest request = new UpdateItemRequest();
         request.setTableName(table);
         request.setKey(serializer.getDynamoHashKey(lease));
-        request.setExpected(serializer.getDynamoLeaseCounterExpectation(lease));
+        request.setExpected(serializer.getDynamoExistantExpectation(lease.getLeaseKey()));
```

Comment:
Why are we replacing leasecounter expectation with leaseKey?

Reply:
Also can we add these two integ tests?

https://github.com/ashwing/amazon-kinesis-client/pull/88/files#diff-d664625d66f92ee877c89352871066abR110

Reply:
> Why are we replacing leasecounter expectation with leaseKey?

This change is fine.